internal/service/file.go

```diff
@@ -7,9 +7,28 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+
+	"github.com/VladPetriv/setup-neovim/pkg/input"
+	"github.com/VladPetriv/setup-neovim/pkg/validation"
 )
 
-func (s service) ExtractAndMoveConfigDirectory(repositoryPath string) error {
+type fileService struct {
+	inputter  input.Inputter
+	validator validation.Validator
+}
+
+func NewFile(inputter input.Inputter, validator validation.Validator) FileService {
+	return &fileService{
+		inputter:  inputter,
+		validator: validator,
+	}
+}
+
+func (f fileService) CheckUtilStatus() map[string]string {
+	return f.validator.ValidateConsoleUtilities()
+}
```

Comment:
Should we remove this method from service and just call `ValidateConsoleUtilities` in `app.go`?

Reply:
Abandoded idea)